PREREQUISITE.md

```diff
@@ -0,0 +1,248 @@
+# Prerequisite
+
+You need to do two actions.
+
+1. Add euphony to your project
+2. Edit your AndroidManifest.xml to permit euphony to record audio.
+
+---
+
+## 1. Add euphony to your project
+
+There are 3 ways adding euphony to your project.
+
+&nbsp;&nbsp;&nbsp;&nbsp;1. Using Maven repository
+
+&nbsp;&nbsp;&nbsp;&nbsp;2. Import the aar/jar file directly
+
+&nbsp;&nbsp;&nbsp;&nbsp;3. Import the Euphony module in your project
+
+### 1.1 Using Maven repository
+
+---
+
+1. Show **_Project window_** with **_Android project view_**. You can select a project view mode in the dropdown list, then open your `build.gradle` file under `Gradle Scripts`.
+
+<img width="392" alt="maven_001_auto_x2_colored_toned" src="https://user-images.githubusercontent.com/27720475/130187144-6bcd31e2-589e-48e0-91ff-d05d28bea4e4.png">
+
+2. Add the following line to the `dependencies` section:
+
+```jsx
```

Comment:
how about ```gradle ?

```gradle
dependencies {
 	// other dependencies
 	// ...
 	implementation 'co.jbear.lib:euphony:0.7.1.6'
 }
```

After that, code highlighting will be worked :)

Reply:
Thanks to your comment, I've learned that code highlighting can be applied for each language!

Reply:
There are some markdown error, because we worked in notion page and it put the codes jsx markdown. So, We need to change codes jsx to gradle or java. There is none jsx codes in here.